src/pybennu/pybennu/providers/python/daemon.py

```diff
@@ -0,0 +1,182 @@
+import argparse
```

Comment:
Why is daemon.py copy-pasted here? Why can't it simply be added to [power_daemon.py](https://github.com/sandialabs/sceptre-bennu/blob/main/src/pybennu/pybennu/providers/power/power_daemon.py) like all the other providers?